Source/JavaScriptCore/llint/WebAssembly.asm

```diff
@@ -885,6 +884,262 @@ end
     break
 end)
 
+# This is the interpreted analogue to WasmBinding.cpp:wasmToWasm
+op(wasm_to_wasm_wrapper_entry, macro()
+    loadq Callee-0x10[sp], ws1
+    bilt ws1, 0, .funcindex
+
+    leap Wasm::Callee::m_indexOrName[ws1], ws1
+    loadq Wasm::IndexOrName::m_indexName[ws1], ws1
+if JSVALUE64
+    const indexTag = (constexpr Wasm::IndexOrName::indexTag)
+else
+    const indexTag = 0
+end
+    andq ~indexTag, ws1
+    jmp .done
+
+.funcindex:
+    noti ws1
+
+.done:
+
+    const ImportFunctionInfoSize = constexpr (sizeof(JSWebAssemblyInstance::ImportFunctionInfo))
+    mulq ImportFunctionInfoSize, ws1
+
+    # Store Callee's wasm callee for import function
+    const ImportFunctionInfoBase = constexpr (sizeof(JSWebAssemblyInstance) + 7) & (~7)
+    leap ImportFunctionInfoBase[wasmInstance], ws0
+    addq ws0, ws1
+    # offsetOfBoxedTargetCalleeLoadLocation
+    loadp JSWebAssemblyInstance::ImportFunctionInfo::boxedTargetCalleeLoadLocation[ws1], ws0
+    loadp [ws0], ws0
+
+    # jit.storeWasmCalleeCallee(scratch, safeCast<int>(sizeof(CallerFrameAndPC)) - safeCast<int>(prologueStackPointerDelta()));
+if JSVALUE64
+    const addressOfCalleeCalleeFromCallerPerspectiveBase = constexpr CallFrameSlot::callee * SlotSize - CallerFrameAndPCSize + PayloadOffset
+    storep ws0, addressOfCalleeCalleeFromCallerPerspectiveBase + CallerFrameAndPCSize - PrologueStackPointerDelta[sp]
```

Comment:
I think you can drop `CallerFrameAndPCSize` since you're just subtracting it then adding it again.